test/e2e/og-routes-custom-font/next.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  outputFileTracingIncludes: {
```

Comment:
It's required to mark the fonts here as they cannot be tracked by nft for deployment

Reply:
Nft can track it fine if it's statically analyzable, this should not be needed